src/handlers/index.ts

```diff
@@ -269,6 +270,44 @@ export async function postBlockPacketHook(ctx: Context, entities: Entities) {
   }
 }
 
+async function postBlockFeeHook(ctx: Context, entities: Entities) {
+  let updatedSendPackets: SendPacket[] = [];
+  let updatedSendPacketFees: SendPacketFeeDeposited[] = [];
+
+  for (let sendPacketFee of entities.sendPacketFees) {
+    let sendPacket = await ctx.store.findOne(SendPacket,
+      {
+        where: {
+          chainId: sendPacketFee.chainId,
+          srcChannelId: sendPacketFee.channelId,
+          sequence: sendPacketFee.sequence
+        }
+      });
+    if (sendPacket) {
+      sendPacket.totalRecvFeesDeposited = sendPacket.totalAckFeesDeposited + BigInt(sendPacketFee.recvGasLimit * sendPacketFee.recvGasPrice);
+      sendPacket.totalAckFeesDeposited = sendPacket.totalAckFeesDeposited + BigInt(sendPacketFee.ackGasLimit * sendPacketFee.ackGasPrice);
+      if (!sendPacket.feesDeposited) {
+        sendPacket.feesDeposited = [sendPacketFee];
+      }
+      // Store up to 20 fee transactions per packet
```

Comment:
Where is 20 coming from?

Reply:
I made it up. Seemed like more fees than anyone would realistically add but also small enough to prevent a single row in the table from getting too bloated. Could probably be as high as 100ish without any problems though.

Reply:
Do we want to store individual fees? What if we just store a sum?

Reply:
@Inkvi let me know what you think of new solution